docs/pipeline_config_documentation/pipeline_ingestion_yml.md

```diff
@@ -0,0 +1,494 @@
+<style>
+  .parameter {
+    border-top: 4px solid lightblue;
+    background-color: rgba(173, 216, 230, 0.2);
+    padding: 4px;
+    display: inline-block;
+    font-weight: bold;
+  }
+</style>
+
+# pipeline.yml for ingestion workflow
+
+## Compute resource options
+
+* <p class="parameter">resources</p>
+  
+    Computing resources to use, specifically the number of threads used for parallel jobs.
+    Specified by the following three parameters:
+
+  - <p class="parameter">threads_high</p> Integer, Default: 1
+        <p>Number of threads used for high intensity computing tasks. 
+        For each thread, there must be enough memory to load all your input files at once and create the MuData object.
+        </p><br>
+  
+  - <p class="parameter">threads_medium</p> Integer, Default: 1
+        <p>Number of threads used for medium intensity computing tasks.
+        For each thread, there must be enough memory to load your mudata and do computationally light tasks.
+        </p><br>
+  
+  - <p class="parameter">threads_low</p> Integer, Default: 1
+  	    <p>Number of threads used for low intensity computing tasks.
+        For each thread, there must be enough memory to load text files and do plotting, requires much less memory than the other two.
+        </p><br>
+
+* <p class="parameter">condaenv</p> String
+  
+    Path to conda environment that should be used to run panpipes.
+    Leave blank if running native or your cluster automatically inherits the login node environment
+
+
+## Loading and concatenating data options
+### Project name and data format
+
+* <p class="parameter">project</p> String, Default: "test"
+    
+    Project name.
+
+
+* <p class="parameter">sample_prefix</p> String, Default: "test"
+    
+    Prefix for sample names.
+
+
+* <p class="parameter">use_existing_h5mu</p> Boolean (True/False)
```

Comment:
Default: False. 
If you want to read an existing MuData object (.h5mu file). set to True. Move the object to the folder where you intend to run the ingestion workflow, and call it ${sample_prefix}_unfilt.h5mu where ${sample_prefix} = sample_prefix argument above.